### PR TITLE
Update hu.m3u

### PR DESCRIPTION
**Pull request by boomski**

### CODE REVIEW

File: channels/hu.m3u [hunk 1]
```diff
@@ -33,6 +33,10 @@ http://cloudfront44.lexanetwork.com:1730/HDE016/hls/livestream.sdp.m3u8
 http://1mstream.digicable.hu/hitmusic/hitmusic.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/64ygAE5.png" group-title="",Hatoscsatorna
 rtmp://lpmedia.hu:1935/Hatoscsatorna/livestream
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/7DtvL6T.jpg" group-title="",Hír TV 
+https://stream.radioszerver.com:443/hirtv/0kTMxkTMxATM.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/7DtvL6T.jpg" group-title="",Hír TV (1 órával késleltetett / 1 Hour Delayed)
+https://stream.radioszerver.com:443/hirtv_delayed/0kTMxkTMxATM.m3u8
```
Comment (freearhey) [inline channels/hu.m3u]: does not work (tested in VLC)

File: channels/hu.m3u [hunk 2]
```diff
@@ -53,3 +59,5 @@ http://188.35.9.5:10007/stolen/246r/parasite.install
 https://cloudfront63.lexanetwork.com:1440/hlsrelay014/hls/livestream.sdp.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/xiSeTRL.jpg" group-title="",Ózdi Városi TV
 https://stream.unrealhosting.hu:443/hls/ozdtv/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/JCHEuWb.jpg" group-title="",VTV Füzesabony
+https://stream.unrealhosting.hu:443/hls/ftv/live.m3u8
```
Comment (freearhey) [inline channels/hu.m3u]: at start-up gives an error "Unsecured connection"

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/hu.m3u (hunk 1 above)
Comment (boomski) [reply channels/hu.m3u]: Indeed They don't work anymore

File: channels/hu.m3u (hunk 2 above)
Comment (boomski) [reply channels/hu.m3u]: For me this link works on VLC , but normally when you open the link you get a message to accept a certificate , and you need to accept it , if you do the link should play.

**Review by freearhey — APPROVED** [review]

**Pull request merged**